src/android.ts

```diff
@@ -139,8 +139,10 @@ export const androidApi = <RunTarget extends SupportedRunTarget<'android'>>(
             }
 
             // Start `frida-server` if it's not already running.
-            const { stdout: fridaCheck } = await python('frida-ps', ['-U'], { reject: false });
+            const { stdout: fridaCheck } = await python('frida-ps', ['-U'], { reject: false, timeout: 10000 });
             if (fridaCheck.includes('frida-server')) return;
+            // Make sure any stuck frida processes are killed.
+            await adbRootShell(['killall', 'frida-server'], { execaOptions: { reject: false } });
```

Comment:
When I encountered this problem, a simple `kill` didn't suffice. I needed `kill -9`.